packages/plugin-node-surrounding-code/surrounding-code.js

```diff
@@ -1,5 +1,8 @@
 const SURROUNDING_LINES = 3
-const { readFile } = require('fs')
+const { createReadStream } = require('fs')
+const { Writable } = require('stream')
+const pump = require('pump')
```

Comment:
this modules is used to pipe streams together. It removes some of the boilerplate around handling errors with more than one stream.